### PR TITLE
Vet clinic database: Create Animals Table

### PR DESCRIPTION
**Pull request by JEFFKAY16**
## Vet clinic database: Create Animals Table

In this update I did the following:
- Created a database
- Created a table
- Inserted the data
- Lastly, I added all commands into corresponding files as instructed

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+
+SELECT name FROM animals WHERE CAST(to_char(date_of_birth, 'YYYY') AS INTEGER) BETWEEN 2016 AND 2019;
+
+SELECT name FROM animals WHERE  neutered IS TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name LIKE 'Agumon' OR name LIKE 'Pikachu';
+
```
Comment (julie-ify) [inline queries.sql]: - [x] The `LIKE` operator is used to match a pattern meaning that this query will still return the date_of_birth of the animal with names `Agumonu` and `Pikachua`. Kindly optimise this query by using a more specific query. hint; 

```sql
SELECT date_of_birth FROM animals WHERE name = 'Pikachu' OR name = 'Agumon';

```

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered IS TRUE;
+
+SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
```
Comment (julie-ify) [inline queries.sql]: - [x] Here also kindly optimize this query to something like this `SELECT * FROM animals WHERE NOT name = 'Gabumon';`

File: schema.sql [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+
+/* Access database */
+
+psql vet_clinic;
+
+/* Created table */
+
 CREATE TABLE animals (
     name varchar(100)
 );
```
Comment (julie-ify) [inline schema.sql]: - [x] Currently, there are two tables named `animals` in your schema.sql file. Kindly remove this one so we can make use of the one that has all the data.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @JEFFKAY16,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Descriptive pull request ✔️ 
- Good code organization ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @JEFFKAY16,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Descriptive pull request ✔️ 
- Good code organization ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (JEFFKAY16) [reply queries.sql]: Done

File: queries.sql (hunk 2 above)
Comment (JEFFKAY16) [reply queries.sql]: Done

File: schema.sql (hunk 3 above)
Comment (JEFFKAY16) [reply schema.sql]: Done

**Review by AshakaE — APPROVED** [review]
Hi @JEFFKAY16 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**